src/components/TextField/commons.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC, MouseEvent } from 'react';
```

Comment:
can we move this to a components folder within the textfield. If we wnt to add more this will gets bigger and is not aligned with our components/file structure

https://orfium.atlassian.net/wiki/spaces/FE/pages/2147057733/Components+File+Structure